fairworkflows/manual_assistant.py

```diff
@@ -0,0 +1,119 @@
+import cgi
+import logging
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from typing import List, Dict, Union
+import base64
+
+from jinja2 import Environment, PackageLoader, select_autoescape
+
+logging.basicConfig(level=logging.INFO)
+env = Environment(loader=PackageLoader('fairworkflows', 'templates'), autoescape=select_autoescape('html'))
+
+# TODO: Remove
+USE_TEST_SERVER = True
+EXAMPLE_STEP_URI = 'http://purl.org/np/RAFszXfE-J3sef_ZX_5LRMM6rHgBt7a1uQH-vZdxfy-RU'
+
+HOST = 'localhost'
```

Comment:
Could we rename these env params to e.g. `MANUALTASK_HOST` etc and put them in `fairworkflows/config.py`
I think that would complete the integration.

Reply:
Added the constants to `config.py`